mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/TaskFactory.java

```diff
@@ -25,7 +25,13 @@
 public interface TaskFactory {
     RuntimeTask getRuntimeTaskInstance(ExecuteStageRequest request, ClassLoader cl);
 
-    UserCodeClassLoader getUserCodeClassLoader(
+    default UserCodeClassLoader getUserCodeClassLoader(
```

Comment:
I'm unsure if this is the right place for this method, given that it doesn't pertain to creating a Task itself. I would maybe move this to some Utils method or have the logic as part of ClassLoaderHandle::createUserCodeClassLoader.

Reply:
sounds good. I will keep a note on this too.